src/features/session_trace/aggregate/index.js

```diff
@@ -93,11 +94,13 @@ export class Aggregate extends AggregateBase {
     registerHandler('trace-jserror', (...args) => this.traceStorage.storeErrorAgg(...args), this.featureName, this.ee)
     registerHandler('pvtAdded', (...args) => this.traceStorage.processPVT(...args), this.featureName, this.ee)
 
-    if (typeof PerformanceNavigationTiming !== 'undefined') {
-      this.traceStorage.storeTiming(globalScope.performance?.getEntriesByType?.('navigation')[0])
-    } else {
-      this.traceStorage.storeTiming(globalScope.performance?.timing, true)
-    }
+    single(() => {
```

Comment:
Asking for my own understanding - are we using `single()` here because it's important to store the timing here instead of, say, in the `if !(this.initialized)` block?

Reply:
I think it could go either place, any strong preference one way vs another?

Reply:
i actually feel like this is a little cleaner without needing an extra import or class attr, so going with this